### PR TITLE
fix: lack of cjs package version

### PR DESCRIPTION
**Pull request by kobyhallx**
# Description

Developer should be able to use both CJS and ESM style packages.

## Problem\*

Resolves Make @noir-lang/noir_js CJS accessible #2821

## Documentation

- [ ] This PR requires documentation updates when merged.

  

  - [ ] I will submit a noir-lang/docs PR.

  

  - [ ] I will request for and support Dev Rel's help in documenting this PR.

  
  

## Additional Context

# PR Checklist\*

- [ ] I have tested the changes locally.
- [ ] I have formatted the changes with [Prettier](https://prettier.io/) and/or `cargo fmt` on default settings.

### CODE REVIEW

File: tooling/noir_js/test/node/cjs.test.cjs [hunk 1]
```diff
@@ -0,0 +1,76 @@
+/* eslint-disable no-undef */
```
Comment (kevaundray) [inline tooling/noir_js/test/node/cjs.test.cjs:1]: Why is this needed?

File: tooling/noir_js/test/node/cjs.test.cjs [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+/* eslint-disable @typescript-eslint/no-var-requires */
+const chai = require('chai');
+const assert_lt_json = require('../noir_compiled_examples/assert_lt/target/assert_lt.json');
+const noirjs = require('@noir-lang/noir_js');
+
+it('001 generates witnesses successfully', async () => {
```
Comment (TomAFrench) [inline tooling/noir_js/test/node/cjs.test.cjs:7]:
```suggestion
it('generates witnesses successfully', async () => {
```

**Review by kevaundray — APPROVED** [review]
